### ISSUE TITLE
Possibility to use it into Mac application

### ISSUE THREAD

**Issue by vojtech-cerveny** [spajak/cef-pdf#19]
Hi,
I wrap your application into my app and I get this error:
```
[1113/143014.498002:ERROR:icu_util.cc(137)] icudtl.dat not found in bundle
[1113/143014.498081:ERROR:icu_util.cc(173)] Invalid file descriptor to ICU data received.
Trace/BPT trap: 5
```
Every file should be present, I copy whole folder into my app. Not sure, I can do it as I do - so that in my `.app` I use your `.app`. Do you have idea, where could a problem?

Did you see it before?

**Comment by spajak** [comment]
Looks like resources from Frameworks cannot be located. I'm afraid not much can be done in this matter. Maybe try to merge cef-pdf instead of wrap.

**Comment by beckyconning** [comment]
Is it possible to link the file more like the linux executable rather than as a macOS `.app` bundle?

**Comment by beckyconning** [comment]
Yeah it is. I'm going to make a pr to make this a single linked executable rather than a bundle.

**Comment by beckyconning** [comment]
you can also copy into application support during installation and call it from there.
